pkg/morph/client/constructor.go

```diff
@@ -82,6 +84,12 @@ func New(key *keys.PrivateKey, opts ...Option) (*Client, error) {
 		panic("empty private key")
 	}
 
+	acc := wallet.NewAccountFromPrivateKey(key)
+	accAddr, err := address.StringToUint160(acc.Address)
```

Comment:
`key.GetScriptHash()` to avoid error check, but I know this can be improved in the `wallet.Account` itself.

Reply:
used `key.GetScriptHash()`